content/learn/book/welcome/app/_index.md

```diff
@@ -0,0 +1,90 @@
++++
+title = "Creating Bevy apps"
+weight = 2
+template = "book-section.html"
+page_template = "book-section.html"
++++
+
+When you're making a game in Bevy, you will have a single `App`, which stores and handles all of your game logic and data.
+Let's make a trivial Hello World app.
+
+```rust
+use bevy::prelude::*;
+
+fn main(){
+  App::build()
+  .add_system(hello.system())
+  .run();
+}
+
+fn hello(){
+    println!("Hello, Bevy!")
+}
+```
+
+So, we first create our [`App`](https://docs.rs/bevy/latest/bevy/app/struct.App.html) by telling it to [`build`](https://docs.rs/bevy/latest/bevy/app/struct.App.html#method.build), performing initialization steps.
+This produces an [`AppBuilder`](https://docs.rs/bevy/latest/bevy/app/struct.AppBuilder.html), which we can use to configure the `App`.
+We do so by add a simple system, which prints "Hello, Bevy!" when it is run.
```

Comment:
```suggestion
We do so by add our `hello()` function as a system through a **trait extension method**, which will print 'Hello, Bevy!' when the app is run.
```
Explaining what `.system()` does is pretty important.

Reply:
Definitely agree. It will likely be fully deprecated for most use cases (including here) though, so this is probably not the best place to describe how the magic works.